tatt/tool.py

```diff
@@ -17,3 +19,15 @@ def idfun(x): return x
             seen[marker] = 1
             result.append(item)
     return result
+
+def get_repo_dir(repodir):
```

Comment:
Please factor this out into a first, independent commit. From our IRC discussion yesterday:

* remove the default ~/gentoo-x86 from the config
* if a repo dir is set in config this should be used, and if it does not exists that should be an error IMHO
* only go to the usual system defaults if the config hasn't set anything